cdc/redo/manager.go

```diff
@@ -566,18 +588,23 @@ func (m *ManagerImpl) bgUpdateLog(
 		}
 	}
 
+	overseerTicker := time.NewTicker(time.Second * 5)
+	defer overseerTicker.Stop()
+	var workTimeSlice time.Duration
+	startToWork := time.Now()
 	for {
 		if len(logs) > 0 || rtsMap.Len() > 0 {
```

Comment:
The content of `if` and `else` branch is very similar, could this logic be simplified further?

Reply:
Yes, we will fix this in anther pr.